via/views/google_drive.py

```diff
@@ -0,0 +1,30 @@
+from pyramid.view import view_config
+
+from via.services.google_drive import GoogleDriveAPI
+from via.services.secure_link import has_secure_url_token
+
+
+@view_config(
+    route_name="google_drive_file",
+    decorator=(has_secure_url_token,),
+)
+def get_file_content(_context, request):
+    """Download a file from Google Drive."""
```

Comment:
You can fit the view config on one line (Black is formatting it onto two lines because of the trailing comma after the last item).

If you're not using `context` in a Pyramid view you can just omit it, you don't need to take a `_context` arg.

Also suggesting a tweak to the route and view name and docstring: I think "proxying" is the verb we tend use for this. I get that this isn't quite the same as what our currently-deployed NGINX-based solution for Google Drive does where it's literally proxying an HTTP response from Google. Still, I think it's close enough, it fills the same functional need as when we "proxy" a PDF from Canvas or Blackboard files or when Via HTML "proxies" a web page etc.

```suggestion
@view_config(route_name="proxy_google_drive_file", decorator=(has_secure_url_token,))
def proxy_google_drive_file(request):
    """Proxy a file from Google Drive."""
```